mbutil/util.py

```diff
@@ -230,10 +236,8 @@ def disk_to_mbtiles(directory_path, mbtiles_file, **kwargs):
                             (?, ?, ?, ?);""",
                             (z, x, y, sqlite3.Binary(file_content)))
                         count = count + 1
-                        if (count % 100) == 0:
-                            for c in msg: sys.stdout.write(chr(8))
-                            msg = "%s tiles inserted (%d tiles/sec)" % (count, count / (time.time() - start_time))
-                            sys.stdout.write(msg)
+                        if (count % 100) == 0 and not silent:
+                            logger.info(" %s tiles inserted (%d tiles/sec)" % (count, count / (time.time() - start_time)))
```

Comment:
If anyone is depending on this output in their scripts, the change will break them. I'm not saying that was a good choice, only looking out for users.

@perrygeo what do you think?

Reply:
Okay, sure.